logoutHandler.js

```diff
@@ -0,0 +1,39 @@
+import AWS from "aws-sdk";
+import util from "util";
+const { promisify } = util;
+
+AWS.config.update({
+  accessKeyId: process.env.AWS_ACC_KEY_ID,
+  secretAccessKey: process.env.AWS_SECRET
+});
+const COGNITO_CLIENT = new AWS.CognitoIdentityServiceProvider({
+  apiVersion: "2016-04-19",
+  region: "us-east-1"
+});
+const utilPromiseAdminLogout = promisify(
+  COGNITO_CLIENT.adminUserGlobalSignOut
+).bind(COGNITO_CLIENT);
+
+export const logout = async (event, context) => {
+  const data = JSON.parse(event.body);
+  //console.log(data);
+  const { email } = data;
+  const userParams = {
+    UserPoolId: process.env.AWS_USER_POOL_ID /* required */,
+    Username: email
```

Comment:
can i logout any user just by passing an email? we should be using the user in the (authenticated) token.